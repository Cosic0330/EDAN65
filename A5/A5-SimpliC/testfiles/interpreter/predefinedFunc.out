New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
7
returning
New get: Variable: return Value: 0
10
returning
New get: Variable: return Value: 0
