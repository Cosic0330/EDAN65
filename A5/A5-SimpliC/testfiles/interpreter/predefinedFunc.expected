New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
7
New get: Variable: return Value: 0
10
New get: Variable: return Value: 0
